packages/quantic/force-app/main/default/lwc/quanticRefineToggle/quanticRefineToggle.css

```diff
@@ -21,6 +21,7 @@
   width: 1.25rem;
   height: 1.25rem;
   font-weight: bold;
+  z-index: 1;
```

Comment:
shouldn't it be higher than 1?

Reply:
the value `1` is already a higher value than the default value attributed to other HTML elements which is the value `auto` so `1` is doing the job here